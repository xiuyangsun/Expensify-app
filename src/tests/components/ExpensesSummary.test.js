import React from 'react';
import { shallow } from 'enzyme';
import { ExpensesSummary} from '../../components/ExpensesSummary';

test('should correctly rendedr ExpensesSummary with 0 expense', () => {
  const wrapper = shallow(<ExpensesSummary expensesCount={0} expensesTotal={0} />);
  expect(wrapper).toMatchSnapshot();
});

test('should correctly rendedr ExpensesSummary with multiple expenses', () => {
  const wrapper = shallow(<ExpensesSummary expensesCount={3} expensesTotal={12345} />);
  expect(wrapper).toMatchSnapshot();
});