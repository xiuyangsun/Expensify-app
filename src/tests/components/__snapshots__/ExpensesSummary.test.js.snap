// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly rendedr ExpensesSummary with 0 expense 1`] = `
<div>
  <h1>
    Viewing 
    0
     
    expenses
     totalling to be 
    $0.00
    .
  </h1>
</div>
`;

exports[`should correctly rendedr ExpensesSummary with multiple expenses 1`] = `
<div>
  <h1>
    Viewing 
    3
     
    expenses
     totalling to be 
    $123.45
    .
  </h1>
</div>
`;
